clusterloader2/api/types.go

```diff
@@ -67,6 +67,8 @@ type Object struct {
 	Basename string `json: basename`
 	// ObjectTemplatePath specifies the path to object definition.
 	ObjectTemplatePath string `json: objectTemplatePath`
+	// Mapping specifies for each placeholder what value should it be replaced with.
+	Mapping map[string]interface{} `json: mapping`
```

Comment:
Can you give some better name than just 'mapping'? Maybe TemplateFillMap?

Reply:
I'm still not convinced if that's a good name as 'mapping' field inside an 'object' struct could mean many different things.  If you `git grep "Mapping"` in the kubernetes code, you'll find several examples of mappings (portMapping, RestMapping, resourceMapping, etc).

Reply:
done